<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AuditReport</class>
 <widget class="QDialog" name="AuditReport">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>715</width>
    <height>505</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Drawings audit report</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,0,0,1">
     <item>
      <widget class="QTabWidget" name="tbSelectParams">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tabSimple">
        <attribute name="title">
         <string>Simple</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>Start time</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="mAuditList">
           <property name="sizeAdjustPolicy">
            <enum>QComboBox::AdjustToContents</enum>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tabAdvanced">
        <attribute name="title">
         <string>Advanced</string>
        </attribute>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="title">
        <string>Parameters</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QRoCheckBox" name="cbPurgeR">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>purge regapps</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRoCheckBox" name="cbPurgeA">
          <property name="text">
           <string>purge all</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRoCheckBox" name="cbExplodeProxy">
          <property name="text">
           <string>explode all proxies</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRoCheckBox" name="cbRemoveProxy">
          <property name="text">
           <string>remove all proxies</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRoCheckBox" name="cbAudit">
          <property name="text">
           <string>audit</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>End time</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="eEndTime">
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>User</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="eUser">
           <property name="text">
            <string/>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>Computer</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="eComputer">
           <property name="minimumSize">
            <size>
             <width>160</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>AutoCAD</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="eAutocad">
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="mGrid">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <column>
      <property name="text">
       <string>ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Code</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Hist.</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Status</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
  <action name="actionPlotView">
   <property name="text">
    <string>View</string>
   </property>
   <property name="toolTip">
    <string>View selected drawing</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QRoCheckBox</class>
   <extends>QCheckBox</extends>
   <header>QRoCheckBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
